### ISSUE TITLE
Is the s-value in ash intended to be compared to q-values?

### ISSUE THREAD

**Issue by kdkorthauer** [pkimes/benchmark-fdr#5]
By 9/22/17, reread the ash paper and weigh in on whether it is appropriate to interpret the s-values as q-values? Preliminary results on simulated data (from @stephaniehicks and @pkimes) show that it is very different than the other FDR benchmark methods.

**Comment by kdkorthauer** [comment]
@pkimes were just looking at the simulations in the ash vignette and noticed a few things that may help us determine what is going on:
1.  When you set pi0 to 1 (all nulls) in the ash vignette, the lsfdr values and s-values get pushed toward 1
2. These simulations directly simulate the effect sizes and standard errors themselves (not the data points)
3. The input to ash is (1) the simulated beta plus random noise and (2) the ground truth standard errors

**Comment by pkimes** [comment]
thanks, @kdkorthauer!
also, to give a little more context, @kdkorthauer was looking at this vignette:
https://github.com/stephens999/ashr/blob/master/vignettes/adaptive_shrinkage.Rmd

and she saw that ASH was performing "as expected" according to the sims in the vignette (vs. the behavior we were seeing in our simulations).
We're still trying to figure out why the vignette's simulations look so different...

**Comment by kdkorthauer** [comment]
I think the difference in the ash vignette simulations and our simulations was that our calculation of SE was assuming that pooled variance for the t-test is equal to the variance estimate ignoring group (see lines 47-48 [here](https://github.com/pkimes/benchmark-fdr/blob/master/simulation_studies/R/simulation-helpers.R)). If we replace that with the standard 2-sample t-test standard error, the lfsr and s-value no longer appear to be severely shrunk toward zero.

tldr: If we give ash underestimated values for SE, it is not able to properly shrink the effect size estimates and makes many false discoveries. I made a branch with the proposed SE estimate change and submitted a pull request.

**Comment by pkimes** [comment]
Just re-ran some simulations, and things look **a lot** better. Thanks @kdkorthauer!
Now I just need to start using @areyesq89's [SummarizedBenchmark](https://github.com/areyesq89/SummarizedBenchmark)!

**Comment by kdkorthauer** [comment]
Great! 

I didn't change the calculation of the `sds` - this is still computed ignoring group - which is used as the independent covariate in some of the other methods. I'm not sure if the other methods assume anything about the type of SD, other than it has to be independent under the null, or if changing it to a pooled calculation would make any difference, but something we should think about.

**Comment by pkimes** [comment]
To be consistent, maybe we should change `sds` too.
I also just checked the papers, and IHW mentions: for the t-test, "the pooled sample variance was used as the covariate." It doesn't look like BL or FDRreg considered sample variance as an independent covariate.

**Comment by pkimes** [comment]
did we ever do anything about this (i.e. changing the `sds` to match `SE`)?

**Comment by kdkorthauer** [comment]
I only changed the SE calculation since it was affecting the ash results. 

I would lean toward keeping the SD calculation as it is in the `du_ttest_sim` function of the `IHWpaper` package since this seems like the covariate they recommend for use as an independent covariate. To be clear on the terminology here, this is an SD ignoring sample group (rather than a 'pooled' calculation which would combine the two within-group SDs).

**Comment by pkimes** [comment]
Makes sense - unfortunate that the wrong wording is used in the IHW paper...
